R/zzz.R

```diff
@@ -1,112 +1,45 @@
-# An internal variable that is set ot the name of the native library depending on its use of the Mono or MS.NET CLR
-nativePkgName <- ""
-
-# An internal variable to buffer startup messages
-startupMsg <- ""
```

Comment:
This was the message on startup: 

> Loading the dynamic library for Microsoft .NET runtime...
Loaded Common Language Runtime version 4.0.30319.42000

I don't think we should by default print this info on load. Runtime version can still be accessed from the env.